plugins/gatsby-remark-args-linker/index.js

```diff
@@ -0,0 +1,74 @@
+const visit = require('unist-util-visit')
+
+const argsRegex = new RegExp(/\-{1,2}[a-zA-Z-]*/, 'ig')
+
+function patch(context, key, value) {
+  if (!context[key]) {
```

Comment:
better to do `in` here? (less questions regarding booleans, etc - I know it's probably less relevant here

Reply:
Sorry, which `in` ?

Reply:
like `key in context` - would that work better and be a bit better defined?

Reply:
Since we are using lodash, the `.has` utility function would be a better implementation.